src/pages/home/sections/landing_top.js

```diff
@@ -158,7 +158,9 @@ const LandingTop = (props) => {
             </Grid>
             <Grid item xs={12} sm={12} md={12} lg={12}>
               <center>
-                <img alt="TechPhantoms" src={LightMode_MLSA_Badge} /><br/><br/>
+                <a href="https://studentambassadors.microsoft.com/"  >
```

Comment:
@imabp Opening external links in a new tab will be great. Put `target='_blank'`. What say?

Reply:
Thank you @vinitshahdeo. Happy to see you back. 

Also @Kp779 you need to use the relation attribute.
`target="_blank"` and `rel = noopener noreferrer`